space-provider/space-framework.md

```diff
@@ -0,0 +1,117 @@
+This document includes the text from the space-framework design document
+
+The full design document can be found here: https://docs.google.com/document/d/1WZCxlGoaCZVymqMhLCa5JZeURdCTzju8C5ABS8V4lW0/edit?usp=sharing
+
+
+# Space support for KubeStellar
+
+## Background
+
+KubeStellar architecture leverages spaces as the container building block for various purposes. For example, the separation between the KubeStellar service provider and the inventory is done by using different spaces for the inventory and the service provider. 
```

Comment:
"container" is potentially confusing here, since the sort involving Linux namespaces and cgroups is so close nearby.

Reply:
I still prefer at this point to work through comments on the google doc until we just move the design doc to here as MD file.